src/libcore/iter/iterator.rs

```diff
@@ -258,6 +258,43 @@ pub trait Iterator {
         None
     }
 
+    /// Returns an iterator adaptor that goes over the original iterator,
+    /// skipping `step` elements at a time.
```

Comment:
Oh, I forgot to change that. I'll update the documentation soon. Thanks for reminding me!